ynr/apps/sopn_parsing/tests/test_extract_page_numbers.py

```diff
@@ -78,6 +78,7 @@ def test_multi_page_sopn_correct_ward_assigning(self):
                     "sopn.pdf", open(example_doc_path, "rb").read()
                 ),
                 source_url="example.com",
+                relevant_pages=posts[post_name],
```

Comment:
Just checking if this change was is intended, as I'd expect this to be set by the management command `sopn_parsing_extract_page_numbers` that gets called below?

Reply:
Michael is right here: this test is to make sure the pages are set after calling the command, so setting them when we set up the test data doesn't make sense.

We want to keep them blank and then check they've been populated after `call_command`.

In doing this, I caught the problem on the previous commit with `headings_are_identical`